src/core/src/op/string_tensor_pack.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "itt.hpp"
+#include "openvino/core/validation_util.hpp"
+#include "openvino/op/op.hpp"
+#include "openvino/op/string_tensor_unpack.hpp"
+#include "string_tensor_pack_shape_inference.hpp"
+
+namespace ov {
+namespace op {
+namespace v15 {
+
+StringTensorPack::StringTensorPack(const Output<Node>& begins, const Output<Node>& ends, const Output<Node>& symbols)
+    : Op({begins, ends, symbols}) {
+    constructor_validate_and_infer_types();
+}
+
+bool StringTensorPack::visit_attributes(ov::AttributeVisitor& visitor) {
+    OV_OP_SCOPE(v15_StringTensorPack_visit_attributes);
+    return true;
+}
```

Comment:
This `override` is not needed.

Reply:
Removing it causes `undefined reference to 'vtable for ov::op::v15::StringTensorPack'` when compiling. I suppose the override needs to be there even if it's functionally the same.

Reply:
Remove it also from class definition in .hpp file.

Reply:
Fixed.